### DISCUSSION TITLE
Inter-relationship between entity types learnt in spaCy

### DISCUSSION THREAD

**Discussion by davesuketu215** [explosion/spaCy#8563] [Help: Coding & Implementations] [answered]
I do understand that spaCy NER model even learns the relationship between the entity types through the embeddings.

In context of the above, I have one question over here:
Would creating a separate model for each entity type degrade the NER performance as compared to having one single model for all the entity types altogether?

Actually for my use-case, one advantage I see in having separate model for each entity type is that I would not have to worry about highly imbalanced dataset, where few entity types are present in miniscule amounts, compared to the rest.

**Comment by polm** [comment, marked as answer]
> Would creating a separate model for each entity type degrade the NER performance as compared to having one single model for all the entity types altogether?

You can try it, but it would probably degrade the performance. 

If you train separate models for each entity you have to figure out how to combine them. Training one model means that the loss function takes care of making the best decision for you.
